data/darktableconfig.xml.in

```diff
@@ -846,6 +846,14 @@
     <longdescription>part of full import path for an import session</longdescription>
   </dtconfig>
 
+  <dtconfig prefs="import" section="session">
+    <name>session/use_filename</name>
+    <type>bool</type>
+    <default>true</default>
```

Comment:
Why not keep current default to avoid any surprise ? This would be a new option for people preferring it but it won't change habits of all current users.

Reply:
done